src/lib/models/abstraction-period.js

```diff
@@ -35,7 +35,7 @@ class AbstractionPeriod extends Model {
 
   set startDay (day) {
     validators.assertDay(day);
-    this._startDay = parseInt(day);
+    this._startDay = parseInt(day) || null;
```

Comment:
Please be aware that this can generate `NaN`. Is that OK?

E.g. if `day` is `cupcakes`, then startDay will be NaN